### PR TITLE
Don't deepcopy just to iterate over dict keys/list items.

### PR DESCRIPTION
**Pull request by DmitryKuzmenko**
Removed 3 unneeded usages of copy.deepcopy().
Related to #27130

### CODE REVIEW

**Comment by cachedout** [comment]
Go Go Jenkins!

**Comment by cachedout** [comment]
Go Go Jenkins!

**Comment by cachedout** [comment]
Go Go Jenkins!

File: salt/state.py [hunk 1]
```diff
@@ -2157,12 +2157,12 @@ def call_high(self, high):
             return errors
         # Compile and verify the raw chunks
         chunks = self.compile_high_data(high)
+        log.critical("CHUNKS: {0}".format(chunks))
```
Comment (s0undt3ch) [inline salt/state.py]: Was this logging message supposed to be committed?

Comment (DmitryKuzmenko) [inline salt/state.py]: Oooops!

Comment (s0undt3ch) [inline salt/state.py]: :smile:

**Pull request merged**